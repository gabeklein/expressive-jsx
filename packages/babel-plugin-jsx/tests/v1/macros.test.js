transform("output attributes", `
  () => {
    bg: red;
    
    <this />
  };
`);

transform("output style of same name", `
  () => {
    marginH: 20;

    <this>Hello World</this>
  };
`)