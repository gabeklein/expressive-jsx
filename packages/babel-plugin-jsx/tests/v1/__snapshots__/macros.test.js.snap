// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`output attributes 1`] = `
  () => <div className="element_7nq" />;
  css(\`
  	.element_7nq {
  		background: red;
  	}
  \`);
`;

exports[`output style of same name 1`] = `
  () => <div className="element_7nq">Hello World</div>;
  css(\`
  	.element_7nq {
  		margin-left: 20px;
  		margin-right: 20px;
  	}
  \`);
`;
